rest/config.go

```diff
@@ -859,13 +877,13 @@ func (dbConfig *DbConfig) validateVersion(ctx context.Context, isEnterpriseEditi
 
 			// validate each collection's config
 			for collectionName, collectionConfig := range scopeConfig.Collections {
-				if isEmpty, err := validateJavascriptFunction(collectionConfig.SyncFn); err != nil {
+				if isEmpty, err := validateJavascriptFunction(jsvm, collectionConfig.SyncFn, 1, 3); err != nil {
 					multiError = multiError.Append(fmt.Errorf("collection %q sync function error: %w", collectionName, err))
 				} else if isEmpty {
 					collectionConfig.SyncFn = nil
 				}
 
-				if isEmpty, err := validateJavascriptFunction(collectionConfig.ImportFilter); err != nil {
+				if isEmpty, err := validateJavascriptFunction(jsvm, collectionConfig.ImportFilter, 1, 99); err != nil {
```

Comment:
you allow 999 args above, and this makes me think you want to put some constants for minJSArgs, maxImportFilterArguments

Reply:
It looks like an import filter only takes one arg, so I'll change the 999 and the 99 to 1.